pkgs/development/libraries/gsignond/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, pkgconfig, meson, ninja, glib, glib-networking, sqlite, gobjectIntrospection, vala_0_34 }:
+
+stdenv.mkDerivation rec {
+  pname = "gsignond";
+  version = "1.0.7";
+
+  name = "${pname}-${version}";
+
```

Comment:
Adding `outputs = [ "out" "lib" "dev" "devdoc" ];` should make the package lighter.